extension/README.md.args.mjs

```diff
@@ -0,0 +1,3 @@
+export const extraContents = `
+testing whoaoaoaoa
```

Comment:
:eyes:

Reply:
Just added the contents! Sorry for the silly mistake